generate_state_database:
  ros__parameters: 
    planning_group: ur_manipulator
    constraints:
      name: serve_coffee  
      constraint_ids: [constraint_1]
      constraint_1:  
        type: orientation
        frame_id: world
        link_name: tool0
        orientation: [0.0, 0.0, 0.0]  
        tolerances: [0.01, 0.01, 3.15]
        weight: 1.0